src/main/java/com/mael/ttt/Mark.java

```diff
@@ -14,4 +15,10 @@
     public String getString() {
         return mark;
     }
+
+    public Mark swapMark() {
+        if (this == EMPTY)
```

Comment:
I've been reading about Optional and I'M SCARED. :scream: 
But I also considered it for this  in the past, so it's already in my ToDo list :+1: